### PR TITLE
Docs updates for v2.0.0

### PR DESCRIPTION
**Pull request by becki-at-luminal**
This PR updates the Regula docs with content for the v2.0.0 release.

### CODE REVIEW

**Review by ameliafugue — APPROVED** [review]
LGTM

File: docs/src/usage.md [hunk 1]
```diff
@@ -430,20 +502,43 @@ Configure which input types Regula should search for:
...
+```
+
+Sync the built-in and custom rules from your [Fugue](https://www.fugue.co) tenant when running Regula locally:
+
+```
+regula init --environment-id a29aec17-ab48-42dc-ade5-c0ba8b650c4c --sync
+```
+
+!!! note
+    If you're using `--sync` in your configuration file, note that the `--sync` flag takes precedence over options that modify the rule set (`--exclude`, `--include`, `--only`, `--no-built-ins`). Those options will be ignored. To use those options with `regula run`, set `--sync=false`:
```
Comment (jason-fugue) [inline docs/src/usage.md:522]:
```suggestion
    If you're using `--sync` in your configuration file, note that the `--sync` flag takes precedence over options that modify the rule set (`--exclude`, `--include`, `--only`, `--no-built-ins`). Those options will be ignored. To override the `--sync` in your configuration file, include `--sync=false` in your `regula run` command:
```

Comment (jason-fugue) [reply docs/src/usage.md:522]: Feel free to ignore or reword if that just makes it more confusing.

**Review by jason-fugue — APPROVED** [review]
Awesome! Thank you for the quick turnaround. I added one very optional suggestion.

File: docs/src/usage.md (hunk 1 above)
Comment (becki-at-luminal) [reply docs/src/usage.md]: Great suggestion, thanks!

**Pull request merged**